### ISSUE TITLE
AsyncAPI Generate Not Working Behind Node Proxy Registry

### ISSUE THREAD

**Issue by sbilello** [asyncapi/generator#1056]
#### Reason/Context
Please try answering few of those questions
- Why we need this improvement?
Developers working behind a proxy
- How will this change help?
It will allow the developer to make it work
- What is the motivation? 
It is a blocker

#### Description
Please try answering few of those questions
- What changes have to be introduced?
Capability to specify the registry mirror
- Will this be a breaking change?
No
- How could it be implemented/designed?
I am not sure

I am trying to follow the tutorial https://killercoda.com/asyncapi/scenario/streetlight-tut. Unfortunately, it does not work if the npm is using a proxy registry.

```
$ asyncapi generate fromTemplate asyncapi.yaml @asyncapi/nodejs-template@0.13.3 -o output -p server=mosquitto
Generation in progress. Keep calm and wait a bit... done

Generator Error: request to https://registry.npmjs.org/@asyncapi%2fgenerator-react-sdk failed, reason: self-signed certificate in certificate chain````

**Comment by derberg** [comment]
Killercode is an external service that is out of our control. We are not aware of any workaround 😞 only one I see is building code docker container, but this one would also have to be downloaded through proxy + it would make a learning experience "custom" and not the best.

Closing for now, and if you have an idea how that could be fixed, please let us know

**Comment by sbilello** [comment]
@Derberg. I followed the tutorial steps in my working station without any docker container.
My working station runs behind a node registry proxy.
I was wondering if the async cli has some parameter that would allow me to avoid
``
$ asyncapi generate fromTemplate asyncapi.yaml @asyncapi/nodejs-template@0.13.3 -o output -p server=mosquitto
Generation in progress. Keep calm and wait a bit... done

Generator Error: request to https://registry.npmjs.org/@asyncapi%2fgenerator-react-sdk failed, reason: self-signed certificate in certificate chain``
 
My current npm works behind a proxy. I tried to set it global but it is still hitting that endpoint.
Is there a way to pass a parameter to the cli to force the npm registry?
